.jenkins/pytorch/multigpu-test.sh

```diff
@@ -28,4 +28,27 @@ time python test/run_test.py --verbose -i distributed/test_c10d_spawn_nccl
 time python test/run_test.py --verbose -i distributed/test_store
 time python test/run_test.py --verbose -i distributed/test_pg_wrapper
 time python test/run_test.py --verbose -i distributed/rpc/cuda/test_tensorpipe_agent
+time python test/run_test.py --verbose -i distributed/_shard/checkpoint/test_checkpoint
```

Comment:
ideally we should figure out a regexp to run all tests automatically  under `distributed/_shard`, but i guess it might be non-trival to make run_test.py aware of that, let's first enable this.

Reply:
cc @janeyx99 We need an automated way of including distributed tests into multigpu tests, was wondering if you had any thoughts here?

Reply:
Ah, could you file an issue with this request? We can consider taking in glob patterns as a run_test.py argument as we already have some functionality to look for test files within the test folder.